# TODO

## Datasets
- A Single-Cell Atlas of In Vivo Mammalian Chromatin Accessibility (cusanovich2018)
- Single cell multi-omics reveal intra-cell-line heterogeneity across human cancer cell lines (zhu2023)
- scPerturb: Harmonized Single-Cell Perturbation Data (peidli2022)
- A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks (parnas2015)
- High-throughput Phenotyping of Lung Cancer Somatic Mutations (berger2018)
- CeDR Atlas: a knowledgebase of cellular drug response (wang2021)

## Models
- Contrastive learning enables rapid mapping to multimodal single-cell atlas of multimillion scale (yang2022)
- Deep transfer learning of cancer drug responses by integrating bulk and single-cell RNA-seq data (chen2022)
- Decoding CAR T cell phenotype using combinatorial signaling motif libraries and machine learning (daniels2022)
- Chromoformer: Learning the histone codes with large genomic windows and three-dimensional chromatin interactions using transformer (lee2022)
- DeepC: predicting 3D genome folding using megabase-scale transfer learning (schwessinger2020)
    - https://www.nature.com/articles/s41592-020-0960-3


## "Best practices"
- A survey of best practices for RNA-seq data analysis (conesa2016)

## Methods
- Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks (trapnell2012)
- Feature selection and dimension reduction for single-cell RNA-Seq based on a multinomial model (townes2019)
- Mapping single-cell data to reference atlases by transfer learning (lotfollahi2022)
- Predicting patient outcomes after treatment with immune checkpoint blockade: A review of biomarkers derived from diverse data modalities (liu2023)
- Diffusion pseudotime robustly reconstructs lineage branching (haghverdi2016)
- SPECTRA
    - https://github.com/dpeerlab/spectra
- scKINETICS: inference of regulatory velocity with single-cell transcriptomics data
    - scKINETICS (Key regulatory Interaction NETwork for Inferring Cell Speed)
    - https://doi.org/10.1093/bioinformatics/btad267
- MAGIC: Recovering Gene Interactions from Single-Cell Data Using Data Diffusion
    - https://doi.org/10.1016/j.cell.2018.05.061

## Benchmarks
- Benchmarking of RNA-sequencing analysis workflows using whole-transcriptome RT-qPCR expression data (everaert2017)
- Power analysis of single-cell RNA-sequencing experiments (svensson2017)
- Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data (freytag2018)

## Tools/Libraries

### Sequencing
- HTSeq: https://htseq.readthedocs.io/en/latest/

### Visualization
- https://pygenometracks.readthedocs.io/en/latest/index.html

### DNA methylation
- ALLCools: ALL methyl-Cytosine tools: https://lhqing.github.io/ALLCools/intro.html

### RNAseq
- Gene/isoform expression: https://github.com/deweylab/RSEM

### Notebooks
- Cellular Genetics group at Sanger Institute: https://cellgeni.readthedocs.io/en/latest/about.html
